vizro-core/src/vizro/static/css/loading.css

```diff
@@ -28,9 +24,6 @@ TODO: If we want to use this class, we need to evaluate if this is the best appr
   width: 100%;
 }
 
-/* Loading spinner */
-
-/* ._dash-loading-callback::after, */
 ._dash-loading::after {
   animation: spinner-border 0.75s linear infinite;
   border: 0.5rem solid lightgrey;
```

Comment:
Very minor, but potentially the `lightgrey` could come from our colour scheme too.